mnist-dp/scripts/requirements.txt

```diff
@@ -0,0 +1,6 @@
+tqdm==4.46.1
+scikit-learn==0.23.1
+tensorflow==1.15.2
```

Comment:
Can we make it work with TF 2.x ?

Reply:
No, the TF-Privacy library isn't compatible for now: https://github.com/tensorflow/privacy#tensorflow-2x

Reply:
Indeed